clang/test/CIR/CodeGen/fullexpr.cpp

```diff
@@ -18,3 +23,31 @@ int go1() {
 // CHECK-NEXT:   cir.yield %[[#RValTmp]] : !s32i
 // CHECK-NEXT: }
 // CHECK-NEXT: cir.store %[[#RVal]], %[[#XAddr]] : !s32i, !cir.ptr<!s32i>
+
+// FLAT: cir.func @_Z3go1v() -> !s32i
+// FLAT: %[[#TmpAddr:]] = cir.alloca !s32i, !cir.ptr<!s32i>, ["ref.tmp0", init] {alignment = 4 : i64}
+// FLAT: %[[#XAddr:]] = cir.alloca !s32i, !cir.ptr<!s32i>, ["x", init] {alignment = 4 : i64}
+// FLAT: cir.br ^[[before_body:.*]]{{ loc.*}}
+// FLAT-NEXT: ^[[before_body]]:  // pred: ^bb0
+// FLAT-NEXT:   %[[#One:]] = cir.const #cir.int<1> : !s32i
+// FLAT-NEXT:   cir.store %[[#One]], %[[#TmpAddr]] : !s32i, !cir.ptr<!s32i>
+// FLAT-NEXT:   %[[#RValTmp:]] = cir.call @_Z2goRKi(%[[#TmpAddr]]) : (!cir.ptr<!s32i>) -> !s32i
+// FLAT-NEXT:   cir.br ^[[continue_block:.*]](%[[#RValTmp]] : !s32i) {{loc.*}}
+// FLAT-NEXT: ^[[continue_block]](%[[#Phi:]]: !s32i {{loc.*}}):  // pred: ^[[before_body]]
```

Comment:
Nit: it's confusing to use `Phi` as the placeholder for a basic block argument IMO, since they're different (although equivalent) forms of SSA representations.

Reply:
Sure. I'll use a name that's more MLIR-like.